response/core/serializers.py

```diff
@@ -63,6 +63,9 @@ class IncidentSerializer(serializers.ModelSerializer):
     comms_channel = CommsChannelSerializer(read_only=True)
     action_items = ActionSerializer(read_only=True, many=True)
 
+    # Weirdly, this ensures we can't unset severity
+    severity = serializers.CharField(required=False)
```

Comment:
This is really counter intuitive 😁 

Is there a reference in the docs that it would be worth linking to? (there may not be!)

Reply:
The documentation isn't very clear.  I figured this out through some stackoverflow answers and a bunch of trail and error through the tests.

https://www.django-rest-framework.org/api-guide/fields/#required

It seems `required=False` means that field doesn't have to be included when the json request is deserialised (including creation), and so it remains unchanged.  By _not_ setting `allow_null` we are still demanding a value is given _if_ it's sent in the json.